src/app/Models/SapReport.php

```diff
@@ -49,40 +49,24 @@ public function account()
 
     /**
      * Generate a user-friendly name for the SAP report file represented by the model.
-     * 
+     *
      * @return string
      * the generated file name
      */
     public function generateDisplayableFileName()
```

Comment:
Just to stick to the same naming convention.

```suggestion
    public function generateReportFileName()
```